src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,18 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.8.1
+
+* Fix Submariner healtcheck support with Globalnet.
+* OVN in kind support deployment support `make clusters using=ovn` (E2E testing & development environment).
+* Support debugging in Libreswan cabledriver.
+* Fix the cabledriver label in the latency metrics.
+* Support non-tls connections for OVN databases.
+* Services can now be recreated without needing to recreate ServiceExport
+* Service Discovery no longer requires Submariner connectivity to be available.
```

Comment:
IMHO this is a bit confusing. We enhanced Service Discovery to only advertise services in Connected clusters.

I hope the message you want to convey is that "Service Discovery no longer requires Submariner to be available"

Reply:
Done.

Reply:
Reading this from an outside perspective still confuses me. Do you mean that Lighthouse can be deployed individually?

Reply:
That it's able to work individually, if you do it via operator, it will work, but subctl support still depends on this https://github.com/submariner-io/submariner-operator/pull/1055  I'll finish it today before moving on to other topics.